docs/wiki/development/windows-provisioning.md

```diff
@@ -1,44 +1,46 @@
 
- * Open a new *Command Prompt*
+ * Open a new *Command Prompt* as an **Administrator** with elevated privileges
  * Execute the following command: `git clone https://github.com/facebook/osquery`
  * Change into the **osquery** root directory: `cd osquery`
- * **As an _Administrator_,** run the batch script to provision a Windows 10 64 bit development environment: `tools\make-win64-dev-env.bat`
- * After completion, create the build folder: `mkdir build\windows10`
+ * Run the batch script to provision a Windows 10 64 bit development environment: `tools\make-win64-dev-env.bat`
+ 
+### Automated Method (recommended)
+
+ * Run the batch script to generate a Visual Studio 2015 solution and commence building the osquery shell, daemon, and tests: `tools\make-win64-binaries.bat`
+
+### Manual Method
+#### Generating the Visual Studio 2015 Win64 Solution
+
+ * Create the build folder: `mkdir build\windows10`
  * Change into the recently created build folder: `cd build\windows10`
  * Generate the Visual Studio 2015 solution files: `cmake ..\.. -G "Visual Studio 14 2015 Win64"`
- * There should be a `OSQUERY.sln` in the build folder. Open this with Visual Studio 2015 that is already installed via the provisioning script.
+ * There should be a `OSQUERY.sln` in the `build\windows10` folder. Open this with Visual Studio 2015.
 
-## Building `osqueryd.exe` and `osqueryi.exe`
+#### Building `osqueryd.exe` and `osqueryi.exe`
  
- * **Automated Process**
-   * Run `tools\make-win64-binaries.bat` from the `osquery` root directory. This will create the CMake build files and execute `cmake --build` to compile the shell and copy all required DLLs into the shell's output directory.
- * **Manual Process**
-   * Open the Visual Studio 2015 solution, `OSQUERY.sln`
-   * Select **Release** or **RelWithDebInfo** as the build configuration.
-   * For `osqueryd.exe`, build the **daemon** project; `osqueryi.exe`, build the **shell** project
-   * After the build succeeds, copy the following DLLs to the directory containing `osqueryd.exe`/`osqueryi.exe` (usually in `build\windows10\osquery\Release` or `build\windows10\osquery\RelWithDebInfo`)
-     * `%ChocolateyInstall%\lib\openssl\local\bin\libeay32.dll`
-     * `%ChocolateyInstall%\lib\openssl\local\bin\ssleay32.dll`
-     * `%ChocolateyInstall%\lib\glog\local\bin\glog.dll`
-     * `%ChocolateyInstall%\lib\linenoise-ng\local\bin\linenoise.dll`
+ * Open the Visual Studio 2015 solution, `OSQUERY.sln`
+ * Select **Release** or **RelWithDebInfo** as the build configuration.
+ * For `osqueryd.exe`, build the **daemon** project; `osqueryi.exe`, build the **shell** project
    
 ## Chocolatey Packages Installed (from official sources)
 
  * chocolatey (if applicable)
  * 7zip.commandline
- * cmake.portable 3.5.0
+ * cmake.portable 3.6.1
  * python2 2.7.11
  * visualstudio2015community (with a custom deployment XML ensuring C/C++ toolchain is installed)
  * thrift 0.9.3 (as a dependency from one of our private packages)
```

Comment:
thrift is actually from chocolatey repo, visualstudio2015community should remain

I think I'll remove the content in the parens to make things simple